data_structures/stack/dynamic_stack.c

```diff
@@ -0,0 +1,244 @@
+/**
+ * @file
+ *
+ * @details
```

Comment:
This might fit more for the brief description, IMO.
```suggestion
 * @brief
```